### PR TITLE
recipe/meta.yaml: bump to 0.4.6, reset build number

### PR DESCRIPTION
**Pull request by piem**

### CODE REVIEW

**Comment by piem** [comment]
hello @conda-forge/core !

any idea why @conda-forge-linter still complains about the selectors syntax?
```
    - pkg-config  # [not win]
    - msinttypes  # [win and py27]
```

seems to me those **do** follow the `<two spaces>#<one space>[<expression>]` pattern.

cheers, piem

File: recipe/meta.yaml [hunk 1]
```diff
@@ -14,21 +14,21 @@ source:
     - avcodec_declfirst.patch  [win and py27]
```
Comment (ocefpaf) [inline recipe/meta.yaml]: @piem it seems that the linter is complaining about this line and not those that you changed.

Comment (piem) [reply recipe/meta.yaml]: oh! thank you @ocefpaf, i missed that!

this patch has been included upstream now, it needs to be removed.

cheers, piem

Comment (ocefpaf) [reply recipe/meta.yaml]: Done!

Comment (piem) [reply recipe/meta.yaml]: thanks! that was fast! I also removed the patch itself.

**Comment by piem** [comment]
@ocefpaf any idea what is going on appveyor?

```
The following packages will be SUPERSEDED by a higher-priority channel:

    conda:            4.3.27-py35h5e3481d_0 defaults    --> 4.3.27-py35_0 conda-forge
    conda-env:        2.6.0-0               defaults    --> 2.6.0-0       conda-forge

ERROR conda.core.link:_execute_actions(337): An error occurred while installing package 'conda-forge::anaconda-client-1.6.5-py_0'.
CondaError: Cannot link a source that does not exist. C:\Miniconda35-x64\Scripts\conda.exe
Attempting to roll back.

CondaError: Cannot link a source that does not exist. C:\Miniconda35-x64\Scripts\conda.exe

Command exited with code 1
```

Could this be related to conda/conda#4374 ?

For info, the last partially successful build was at 78379965. Partially because of conda-forge/ffmpeg-feedstock#6 and conda-forge/ffmpeg-feedstock#25.

**Comment by ocefpaf** [comment]
See https://gitter.im/conda-forge/conda-forge.github.io?at=59d7880b177fb9fe7e4782a0

I guess we need to fix `anaconda-client-1.6.5-py_0`, hold on.

**Comment by ocefpaf** [comment]
@conda-forge-admin, please rerender

**Comment by isuruf** [comment]
@conda-forge-admin, please rerender. This should fix your issue

**Comment by ocefpaf** [comment]
@isuruf want to bot race :wink:

**Comment by piem** [comment]
thank you @ocefpaf and @isuruf, looking much better now!

i'll just remove ffmpeg on `(win and py27)` for now, and we should be good to go.

**Comment by piem** [comment]
> i'll just remove ffmpeg on `(win and py27)` for now, and we should be good to go.

Except that not, because of this [hard-coded check for conda on windows](https://github.com/aubio/aubio/blob/master/python/lib/moresetuptools.py#L184-L185).

This workaround was required due to the lack of `pkg-config` on windows (see conda-forge/pkg-config-feedstock#11).

Now, i wonder if something like this would work to build against libav only if `pyver > 2.7`?

```diff
diff --git a/python/lib/moresetuptools.py b/python/lib/moresetuptools.py
index 2f621a7c..30bf9be5 100644
--- a/python/lib/moresetuptools.py
+++ b/python/lib/moresetuptools.py
@@ -182,7 +182,7 @@ class build_ext(_build_ext):
                 add_external_deps(extension, usedouble=enable_double)
                 # force adding libav on windows
                 if os.name == 'nt' and ('WITH_LIBAV' in os.environ \
-                        or 'CONDA_PREFIX' in os.environ):
+                        or ('CONDA_PREFIX' in os.environ and os.environ['CONDA_PY'] != '27')):
                     add_libav_on_win(extension)
                 # add libaubio sources and look for optional deps with pkg-config
                 add_local_aubio_sources(extension)
```

**Comment by piem** [comment]
arg, i pushed by mistake to conda-forge/aubio-feedstock instead of pushing to my own branch :-(

**Pull request merged**